scipy/stats/_continuous_distns.py

```diff
@@ -3282,6 +3283,308 @@ def _ppf(self, q, k, s):
 mielke = mielke_gen(a=0.0, name='mielke')
 
 
+class kappa4_gen(rv_continuous):
+    """Kappa 4 parameter distribution.
+
+    %(before_notes)s
+
+    Notes
+    -----
+    The probability density function for kappa4 is::
+
+        kappa4.pdf(x, h, k) = (1.0 - k*x)**(1.0/k - 1.0)*
+                              (1.0 - h*(1.0 - k*x)**(1.0/k))**(1.0/h-1)
+
+    if ``h`` and ``k`` are not equal to 0.
+
+    If ``h`` or ``k`` are zero then the pdf can be simplified:
+
+    h = 0 and k != 0::
+
+        kappa4.pdf(x, h, k) = (1.0 - k*x)**(1.0/k - 1.0)*
+                              exp(-(1.0 - k*x)**(1.0/k))
+
+    h != 0 and k = 0::
+
+        kappa4.pdf(x, h, k) = exp(-x)*(1.0 - h*exp(-x))**(1.0/h - 1.0)
+
+    h = 0 and k = 0::
+
+        kappa4.pdf(x, h, k) = exp(-x)*exp(-exp(-x))
+
+    kappa4 takes ``h`` and ``k`` as shape parameters.
+
+    The kappa4 distribution returns other distributions when certain
+    ``h`` and ``k`` values are used.
+
+    +------+-------------+----------------+---------------+
+    | h    | k=0.0       | k=1.0          | -inf<=k<=inf  |
+    +======+=============+================+===============+
+    | -1.0 | Logistic    |                | Generalized   |
+    |      |             |                | Logistic(1)   |
+    +------+-------------+----------------+---------------+
+    |  0.0 | Gumbel      | Reverse        | Generalized   |
+    |      |             | Exponential(2) | Extreme Value |
+    +------+-------------+----------------+---------------+
+    |  1.0 | Exponential | Uniform        | Generalized   |
+    |      |             |                | Pareto        |
+    +------+-------------+----------------+---------------+
+
+    (1) The "Generalized Logisitic" test doesn't pass.  There are at
+        least five generalized logistic distributions.  Four are
+        described here:
+        https://en.wikipedia.org/wiki/Generalized_logistic_distribution
+        The "fifth" one is the one kappa4 should match which isn't
+        implemented in scipy:
+        https://en.wikipedia.org/wiki/Talk:Generalized_logistic_distribution
+        http://www.mathwave.com/help/easyfit/html/analyses/distributions/gen_logistic.html
```

Comment:
OK, I see now. This is some other definition of genlogistic. Maybe best to slightly reword the comment though. "Test does not pass" is not the best wording for the docstring. IMO it's better to just state that what's implemented as `genlogistic` in scipy is not what's meant by `genlogistic` in the literature. You have further links to the literature, great.